src/common/hostPlatform.ts

```diff
@@ -88,6 +96,10 @@ class OSXHostPlatform extends UnixHostPlatform {
     public getPlatformId(): HostPlatformId {
         return HostPlatformId.OSX;
     }
+
+    public getTargetPlatformCompatibility(targetPlatformId: TargetPlatformId): boolean {
+        return targetPlatformId !== TargetPlatformId.INVALID;
```

Comment:
I would replace this with `targetPlatformId === TargetPlatformId.ANDROID || targetPlatformId === TargetPlatformId.IOS` since we might support other platforms in the future.